onnx/onnx.in.proto

```diff
@@ -71,17 +71,20 @@ enum Version {
   //    - Added new message OperatorSetIdProto
   //    - Added opset_import in ModelProto
   // - For vendor extensions, added domain in NodeProto
-  IR_VERSION = 0x00000003;
+  IR_VERSION_2017_11_3 = 0x00000003;
+  
+  // IR VERSION 1.0.0 published on Jan 23, 2018
+  // Merged onnx.proto with onnx-ml.proto.
+  // Removed Sequence and Map type for onnx ml.
+  // Added Abstract for any opaque type.
+  IR_VERSION = 0x01000000;
```

Comment:
So... this may have been a breaking change for onnx-ml, but it's not a breaking one for onnx, is it?